src/app/globals.css

```
@@ -28,31 +34,169 @@
   }
 
   .divider {
-    @apply h-0.5 w-full rounded;
+    @apply h-px w-full rounded-full bg-[#3A3E48];
   }
 
+  /* Kill browser / @tailwindcss/forms default blue accent & focus ring */
   .checkbox {
-    @apply border-orange bg-lightOrange h-6 w-6 rounded-sm border;
+    @apply h-6 w-6 cursor-pointer rounded-md border;
+    appearance: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)